baseband/mark5b/base.py

```diff
@@ -172,12 +204,14 @@ class Mark5BStreamReader(VLBIStreamReaderBase, Mark5BFileReader):
 
     _frame_class = Mark5BFrame
 
-    def __init__(self, fh_raw, nchan, bps=2, ref_mjd=None, thread_ids=None,
-                 frames_per_second=None, sample_rate=None, squeeze=True):
+    def __init__(self, fh_raw, nchan, bps=2, kday=None, ref_time=None,
+                 thread_ids=None, frames_per_second=None, sample_rate=None,
+                 squeeze=True):
         # Pre-set fh_raw, so FileReader methods work
         # TODO: move this to StreamReaderBase?
         self.fh_raw = fh_raw
-        self._frame = self.read_frame(ref_mjd=ref_mjd, nchan=nchan, bps=bps)
+        self._frame = self.read_frame(ref_time=ref_time, kday=kday,
```

Comment:
Let's swap the order in the call too (though it doesn't really matter with keywords, any lessening of cognitive dissonance is good).